src/app/api/webhook/level/route.ts

```diff
@@ -12,8 +14,10 @@ export async function POST(req: Request) {
     return NextResponse.json({ status: 'unknown' });
   }
 
-  const body = await req.json();
+  let body = await req.json();
```

Comment:
you actually can assign this to be `const` since you aren't not reassigning a new value to body. You are manipulating body's property, not reassignment